index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Coding Quiz</title>
+</head>
+<body>
+ <header>
+  <section class="head">
+    <h1>&lt Coding Quiz &gt</Coding></h1>
+  </section>
```

Comment:
I see why you've used this `section`, but since it's only to center the headline, you could remove it and give `text-align: center` to the `h1` directly.